### DISCUSSION TITLE
how to disable number shortcut

### DISCUSSION THREAD

**Discussion by kzhui125** [ohmyzsh/ohmyzsh#12158] [Help] [answered]
zsh:

<img width="249" alt="image" src="https://github.com/ohmyzsh/ohmyzsh/assets/11925432/4e58f33d-7c8c-4e83-91e8-47a9e92edaed">

bash:

```sh
bash: 0: command not found
```

**Comment by mcornella** [comment, marked as answer]
These aliases are defined in `lib/directories.zsh`. You can follow the docs in [Customization](https://github.com/ohmyzsh/ohmyzsh/wiki/Customization) to override the lib file and put your custom content there.

You can also simply paste the line below in your .zshrc file at the very bottom, which will remove only these aliases:
```sh
unalias -m '[0-9]'
```
